ext/intl/tests/dateformat_format_relative.phpt

```diff
@@ -0,0 +1,59 @@
+--TEST--
+datefmt_format_code() with relative formats
+--SKIPIF--
+<?php if (!extension_loaded("intl")) print "skip"; ?>
+--FILE--
+<?php
+
+printFormat(IntlDateFormatter::RELATIVE_FULL, IntlDateFormatter::NONE, getYesterday());
+printFormat(IntlDateFormatter::RELATIVE_LONG, IntlDateFormatter::NONE, getYesterday());
+printFormat(IntlDateFormatter::RELATIVE_MEDIUM, IntlDateFormatter::NONE, getYesterday());
+printFormat(IntlDateFormatter::RELATIVE_SHORT, IntlDateFormatter::NONE, getYesterday());
```

Comment:
In these tests the output for all the RELATIVE_ constants is the same. Is there a way to produce different output?

Reply:
I was also surprised by the fact that they all are the same. I'll have a more thorough look about their difference.

Reply:
I found the definitive answer to your question at http://userguide.icu-project.org/formatparse/datetime

>  As currently implemented, relative date formatting only affects the formatting of dates within a limited range of calendar days before or after the current date, based on the CLDR <field type="day">/<relative> data: For example, in English, "Yesterday", "Today", and "Tomorrow". Within this range, the specific relative style currently makes no difference.

Reply:
You can add one more test for a case that doesn't use relative printing, so we should see a difference there.

Reply:
Done! Works as expected locally.